management/example_test.go

```diff
@@ -134,6 +142,7 @@ func ExampleConnectionManager_List() {
 	)
 	if err != nil {
 		// handle err
+		return
```

Comment:
Just in case we encounter errors we should return to not have the examples fail.